README.md

```diff
@@ -1,13 +1,15 @@
-# Steampipe Table Dump (spdump)
+# Steampipe Exporter
 
-This repository contains a tool for data dumping from Steampipe plugins. It allows you to query and export data from Steampipe plugins.
+A Steampipe exporter fetches data from cloud services and APIs. Each exporter is a standalone binary that extracts data using a Steampipe plugin. These repository enables you to build an exporter derived from a [Steampipe plugin](https://hub.steampipe.io/plugins).
 
 ## Getting Started
 
-To get started with the Steampipe Data Dump tool, you'll need to build and install it. You can do so using the provided `Makefile`. Run the following command to build the tool and install it in the specified directory (default is `/usr/local/bin`):
+If you just want to acquire and run the binary for an exporter, you can download an installer from [Steampipe downloads](https://steampipe.io/downloads). See the [installation docs](https://turbot.com/docs/steampipe_export/install) for details. 
```

Comment:
Please include the how to install step here as well ... no need to push them to the downloads page, which makes it feel more "corporate".

It's much simpler now with the redirect URL available.